package.json

```diff
@@ -54,8 +54,12 @@
     "resumer": "0.0.0",
     "rollup": "^0.55.1",
     "rollup-plugin-babel": "^3.0.3",
+    "string-to-arraybuffer": "^1.0.0",
```

Comment:
I took a look at [string-to-arraybuffer](https://github.com/dfcreative/string-to-arraybuffer) at what it dose, think it can lead to unintentional behavior since it dose different things depending on what kind of string it is (dataUri, base64 or plain strings)

What if the server only accepts base64 strings? the lib will convert it to none-base64 buffer...?
can't we just use `Buffer.from(str, 'utf-8')`?